registry/quilt_server/prod_config.py

```diff
@@ -10,7 +10,11 @@
 REGISTRY_URL = os.environ['REGISTRY_URL']
 CATALOG_HOST = os.environ['CATALOG_HOST']
 
-CATALOG_URL = 'https://%s' % CATALOG_HOST
+CATALOG_URL = os.environ.get('CATALOG_URL', 'https://%s' % CATALOG_HOST)
+if not CATALOG_URL.startswith("https"):
+    print("WARNING: INSECURE CONNECTION TO CATALOG")
+    # require verbose environment variable to be defined
+    ALLOW_INSECURE_CATALOG_ACCESS = os.environ['ALLOW_INSECURE_CATALOG_ACCESS']
```

Comment:
Just make it an assert or something similar? No point in assigning it to a variable if you only need to check that it is defined.